src/interfaces/IConditionalOrder.sol

```diff
@@ -14,6 +14,16 @@ interface IConditionalOrder {
     ///      A parameter of `string` type is included to allow the caller to specify the reason for the failure.
     error OrderNotValid(string);
 
+    // --- errors specific for polling
+    // Signal to a watch tower that polling should be attempted again.
+    error PollTryNextBlock(string reason);
```

Comment:
I wonder if this information should/could already be emitted when the Conditional Order is created?

Reply:
Great idea, and certainly valid for single order types. I've added the issue #75 to track this initiative. Worst case scenario in the current scheme, we consume one additional poll cycle on each conditional order creation. I think it would be reasonable for us to defer this to avoid widening the scope too much for the valory watch-tower adaption (would necessitate monitoring two events, given the TWAP legacy).